app/src/main/java/com/stevesoltys/seedvault/transport/backup/PackageService.kt

```diff
@@ -22,13 +23,15 @@ private val TAG = PackageService::class.java.simpleName
 
 private const val LOG_MAX_PACKAGES = 100
 
+// This must be kept in sync with frameworks/base/services/backup/java/com/android/server/backup/utils/BackupEligibilityRules.java
```

Comment:
Maybe use cs.android.com links (pointing to a tag preferably) so we can easily click

That's what we do in some other places.